config/services.yaml

```diff
@@ -34,6 +37,10 @@ services:
             - !tagged_iterator app.feed.feed_type
             - '@feeds.cache'
 
+    App\Service\AuthScreenService:
+        arguments:
+            - '@authscreen.cache'
```

Comment:
Not needed if you follow the rules for autowireable naming (https://symfony.com/doc/current/cache.html#creating-custom-namespaced-pools)